src/tty/windows.rs

```diff
@@ -179,6 +179,17 @@ impl RawReader for ConsoleRawReader {
                             KeyPress::Down
                         });
                     }
+                    winuser::VK_BACK => {
+                        return Ok(if ctrl {
+                            KeyPress::ControlBackspace
+                        } else if shift {
+                            KeyPress::ShiftBackspace
+                        } else if meta {
+                            KeyPress::MetaBackspace
+                        } else {
+                            KeyPress::Backspace
+                        });
+                    }
```

Comment:
These lines are partially redundant with lines 235:236 and 247:248. So I would prefer you (re)move this block.

Reply:
understood.